### PR TITLE
Documentation for Parallel Categories trace type

### PR DESCRIPTION
**Pull request by jonmmease**
### Overview
This PR contains WIP documentation for the still-under-development Parallel Categories trace type (https://github.com/plotly/plotly.js/pull/2963).  This trace is currently slated for plotly.js 1.42, and I'd like to have JavaScript and Python docs ready to go soon after release.

 - [x] JavaScript
 - [x] Python example notebook

TODO:
 - [x] Move CodePen examples to plotly account
 - [x] Remap plotly.js `src` URLs to official CDN after 1.42 is released

### Highlights
screen shot 2018-09-19 at 9 17 12 am

### CodePen Examples
https://codepen.io/plotly/pen/KrpmQO
https://codepen.io/plotly/pen/yQNbxy
https://codepen.io/plotly/pen/BGNRqM
https://codepen.io/plotly/pen/jQPmXN
https://codepen.io/plotly/pen/EOjmrW

### CODE REVIEW

**Comment by jonmmease** [comment]
### Documentation GIFs
parcats-titanic

cars-brushing

cars-multicolor-brushing

**Comment by jonmmease** [comment]
Ready for review 🎉 
cc @cldougl

File: _posts/python/statistical/parcats/2015-06-30-parcats.html [hunk 1]
```diff
@@ -0,0 +1,596 @@
...
+    
+<span class="c1"># Register callback on scatter selection...</span>
+<span class="n">fig</span><span class="o">.</span><span class="n">data</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span><span class="o">.</span><span class="n">on_selection</span><span class="p">(</span><span class="n">update_color</span><span class="p">)</span>
+<span class="c1"># and parcats click</span>
+<span class="n">fig</span><span class="o">.</span><span class="n">data</span><span class="p">[</span><span class="mi">1</span><span class="p">]</span><span class="o">.</span><span class="n">on_click</span><span class="p">(</span><span class="n">update_color</span><span class="p">)</span>
+
+<span class="c1"># Display figure</span>
+<span class="n">widgets</span><span class="o">.</span><span class="n">VBox</span><span class="p">([</span><span class="n">color_toggle</span><span class="p">,</span> <span class="n">fig</span><span class="p">])</span>
```
Comment (cldougl) [inline _posts/python/statistical/parcats/2015-06-30-parcats.html]: such an awesome example!

File: _posts/python/statistical/parcats/2015-06-30-parcats.html [hunk 2]
```diff
@@ -0,0 +1,596 @@
...
+<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
+</div>
+<div class="inner_cell">
+<div class="text_cell_render border-box-sizing rendered_html">
+<p><img src="https://user-images.githubusercontent.com/15064365/48087055-4e6acc00-e1cc-11e8-8f0c-c8316eb4def2.gif" alt="Multi-color brushing"></p>
+
+</div>
+</div>
+</div>
+
```
Comment (cldougl) [inline _posts/python/statistical/parcats/2015-06-30-parcats.html]: Add reference section to point to reference page -- similar to https://plot.ly/python/parallel-coordinates-plot/#reference

File: _posts/python/statistical/parcats/2015-06-30-parcats.html [hunk 3]
```diff
@@ -0,0 +1,596 @@
+---
+permalink: python/parcats/
```
Comment (cldougl) [inline _posts/python/statistical/parcats/2015-06-30-parcats.html]: same url comment as js

File: _posts/python/statistical/parcats/2015-06-30-parcats.html [hunk 4]
```diff
@@ -0,0 +1,596 @@
...
+description: How to make parallel categories visualizations in Python with Plotly.
+name: Parallel Categories
```
Comment (cldougl) [inline _posts/python/statistical/parcats/2015-06-30-parcats.html]: Parallel Categories Diagram

File: _posts/python/statistical/parcats/2015-06-30-parcats.html [hunk 5]
```diff
@@ -0,0 +1,596 @@
+---
+permalink: python/parcats/
+description: How to make parallel categories visualizations in Python with Plotly.
```
Comment (cldougl) [inline _posts/python/statistical/parcats/2015-06-30-parcats.html]: I'd call this diagrams as well to keep it consistent w/ the js doc

**Comment by jonmmease** [comment]
Thanks for the review @cldougl! I implemented all of your suggestions in 43aa838.  Merging!

**Pull request merged**